Add patch to move statusbar clock back to the right hand side.

dothebakedpotato opened the pull request:
This patch moves the status bar clock back into the system status icon area on the right hand side, as in Android versions <9.0.

Tested on a Pixel 2 XL. It should be good for all Pixels without a notch.

Commit:
Move statusbar clock back to the right hand side

Moves statusbar clock back to the right hand side and removes support for the notch. Should only be applied for pixels without a notch.

Commit:
Update manifest for patch 00005-move-statusbar-clock-back-to-rhs

Commit:
Fix some layout issues.

Commit:
Fix mismatched tag.

Commit:
Fix mismatched tag.

Commit:
Prevent clock being hidden during heads up notifications.

Commit:
Update paths.

dan-v commented:
@dothebakedpotato - thanks for this contribution! I'm still trying to decide how I want to handle it though. The existing patches here are all privacy/security focused and this is a cosmetic change. Maybe we could create a sub directory here called 'cosmetic' (or something like that), just to keep the two separate. Thoughts?

Commit:
Add patch to enable wifi display mirroring and casting.

ghost commented:
@dan-v  Hi Dan,
Yes, fair point. I was in two minds about whether to submit it, but I figured that there might be some other people out there who were as triggered by google's design decision as I was.

Don't mind if you want to put it in a directory. Another idea is maybe to demarcate the numbering of the files somehow, e.g.

0000 - 1999    Security / privacy patches
2000 - 3999    Functional patches
4000 - 5999    Aesthetic patches

Cheers.

Commit:
Delete 00005-move-statusbar-clock-back-to-rhs.patch

Not working yet.

47Ohms committed:
Revert "Delete 00005-move-statusbar-clock-back-to-rhs.patch"

This reverts commit 353fd2f3a99ec7bd8d6eae9da83597c018df5fe3.

47Ohms committed:
Revert "Add patch to enable wifi display mirroring and casting."

Not yet working.

47Ohms committed:
Add patch to enable WiFi and 4G calling.

47Ohms committed:
Add enables to device system.prop

47Ohms committed:
Fix incorrect paths.

47Ohms committed:
Disabled VOIP over IMS mode. Causes problems with carrier.

47Ohms committed:
Enable WMR-WB for HD voice.

47Ohms committed:
Now AMR-WB is enabled, try disabling VOIP over IMS mode again

dan-v commented:
That's not a bad idea @dothebakedpotato. I'm open to either the numbering scheme you mentioned or folder subdirectories. Let's say the clock change falls under aesthetic changes. For the other change for wifi/4g calling, let's make that a different category like carrier/device specific patches since it's hardcoded to wahoo kernel at the moment. 

In the future, I'd like to come up with some better method for sharing and applying these patches. This was just the quick and dirty first attempt to allow people to share tweaks and such. As a contributor if you have any ideas on this front I'm all ears.

47Ohms committed:
Increase Binder Proxy counting thresholds to test misbehaving app.

47Ohms committed:
Use cloudflare as default dns provider instead of google.

47Ohms committed:
Add patch to disable SUPL assisted GPS, which is known to leak information to
the provider (Google by default. See https://blog.wirelessmoves.com/2014/08/supl-reveals-my-identity-and-location-to-google.html)

47Ohms committed:
Fix patch.

47Ohms committed:
Fix patch

47Ohms committed:
Remove index line

47Ohms committed:
Fix patch

47Ohms committed:
Was on wrong branch (duh)
